api-tests/src/utils/jar-generator.ts

```diff
@@ -29,6 +30,7 @@ export const generateJar = async (
   const payload = {
     ...payloadData,
     ...{
+      reprove_identity: reproveIdentity,
```

Comment:
Note that `false` and `undefined` are not (necessarily) equivalent from a JSON POV - but we should be treating them the same in core